frontend/packages/metal3-plugin/OWNERS

```diff
@@ -0,0 +1,11 @@
+reviewers:
+  - jtomasek
+  - honza
+  - flofuchs
+  - knowncitizen
+  - dantrainor
+approvers:
+  - jtomasek
+  - honza
+  - flofuchs
+  
```

Comment:
nit: extra newline

Reply:
ugh, @vojtechszocs suggested adding it in the last comment :) https://github.com/openshift/console/pull/1539#discussion_r289427223

Reply:
Done